mlir/lib/Dialect/XeGPU/IR/XeGPUOps.cpp

```diff
@@ -110,6 +110,65 @@ isValidGatherScatterParams(Type maskTy, VectorType valueTy,
   return success();
 }
 
+static LogicalResult
+isValidGatherScatterMemRefParams(Type maskTy, VectorType valueTy,
```

Comment:
Today `create_tdesc` is limited to 1D memref.  Should that apply here too? If not what does it mean to have nD memref? (I can guess but I shouldn't have to 😉)